src/main/scala/uk/gov/nationalarchives/aggregate/processing/modules/TransferOrchestration.scala

```diff
@@ -104,6 +126,7 @@ object TransferOrchestration {
 
   trait StepFunctionInput {}
```

Comment:
Should extend the trait to enforce sub types having the consignment id:
`trait StepFunctionInput { def consignmentId: String }`